G/GTK3/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg.BinaryPlatforms
+
+name = "GTK3"
+version = v"3.24.11"
+
+# Collection of sources required to build GTK
+sources = [
+    "http://ftp.gnome.org/pub/gnome/sources/gtk+/$(version.major).$(version.minor)/gtk+-$(version).tar.xz" =>
+    "dba7658d0a2e1bfad8260f5210ca02988f233d1d86edacb95eceed7eca982895",
+    "./bundled"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gtk+-*/
+
+# We need to run some commands with a native Glib
+apk add glib-dev gtk+3.0
+# This is awful, I know
+ln -sf /usr/bin/glib-compile-resources ${prefix}/bin/glib-compile-resources
+ln -sf /usr/bin/glib-compile-schemas ${prefix}/bin/glib-compile-schemas
+ln -sf /usr/bin/gdk-pixbuf-pixdata ${prefix}/bin/gdk-pixbuf-pixdata
```

Comment:
Just a quick note that you'll need to remove these after `make install` so that they don't accidentally get picked up.